.github/workflows/ci.yml

```diff
@@ -15,6 +15,15 @@ jobs:
         with:
           exclude-dirs: portable
 
+  port-formatting:
```

Comment:
>The use of additional ASM files allows this port to easily be formatted since there is no inline ASM in the .c and .h files

I'm a big fan of this too. Keeps the assembly centrally located.